subsys/caf/modules/sensor_sampler.c

```diff
@@ -51,7 +52,8 @@ static void update_sensor_state(const struct sensor_config *sc, struct sensor_da
 	EVENT_SUBMIT(event);
 }
 
-static void send_sensor_event(const char *descr, const float *data, const size_t data_cnt)
+static void send_sensor_event(const char *descr, const float *data, const size_t data_cnt,
+			      atomic_t *active_sensor_events_cnt)
```

Comment:
s/active_sensor_events_cnt/event_cnt

please use buffer_count, data_count, etc intead of buffers_count.., etc

please use shortest possible names (C is a spartan language) - see https://www.kernel.org/doc/html/v4.10/process/coding-style.html
In this case this is a local function and so does not need a very expressive names for arguments